crt_portal/static/sass/_uswds-theme-custom-styles.scss

```diff
@@ -258,6 +261,40 @@ form#report-form {
   }
 }
 
+.tr-status-new {
+  td {
+    font-weight: bold;
+  }
+}
+
+// crt view statuses
+.status-new {
+  color: $white !important;
+  background-color: $blue-warm-vivid-70;
+  padding: 4px;
+  padding-left: 10px;
+  padding-right: 10px;
+  border-radius: 8.5rem;
```

Comment:
Hmmm ... `8.5rem` == `136px`, which is too high for a `border-radius`. 

I would re-check the intended `border-radius` with Aviva, it looks fairly small and subtle in the mockup.